server/models/Collective.js

```diff
@@ -276,6 +276,8 @@ export default function (Sequelize, DataTypes) {
 
       expensePolicy: DataTypes.TEXT, // markdown
 
+      contributionPolicy: DataTypes.TEXT, // markdown
```

Comment:
Maybe we should limit the size?

Reply:
With varchar?

Reply:
Just with a sequelize validation like https://github.com/opencollective/opencollective-api/blob/cc3f8d8946559d3c62140b103c9f3ce4d392ee89/server/models/Collective.js#L199, changing the type is more complex to update if we need to in the future